src/pages/Compare/index.spec.tsx

```diff
@@ -324,6 +356,106 @@ describe(__filename, () => {
     });
   });
 
+  it('dispatches fetchVersionFile() on mount', () => {
+    const addonId = 9999;
+    const baseVersionId = 1;
+    const headVersionId = baseVersionId + 1;
+    const version = { ...fakeVersionWithDiff, id: headVersionId };
+    const store = configureStore();
+
+    store.dispatch(
+      versionsActions.loadVersionInfo({
+        version,
+        comparedToVersionId: baseVersionId,
+      }),
+    );
+
+    store.dispatch(
+      versionsActions.loadDiff({
+        addonId,
+        baseVersionId,
+        headVersionId,
+        version,
+      }),
+    );
+
+    const dispatch = spyOn(store, 'dispatch');
+    const fakeThunk = createFakeThunk();
+    const _fetchVersionFile = fakeThunk.createThunk;
+
+    render({
+      ...getRouteParams({ addonId, baseVersionId, headVersionId }),
+      _fetchVersionFile,
+      store,
+    });
+
+    expect(dispatch).toHaveBeenCalledWith(fakeThunk.thunk);
+    expect(_fetchVersionFile).toHaveBeenCalledWith({
+      addonId,
+      versionId: version.id,
+      path: version.file.selected_file,
+    });
+  });
+
+  it('does not dispatch fetchVersionFile() when a file is loading', () => {
+    const addonId = 9999;
+    const baseVersionId = 1;
+    const headVersionId = baseVersionId + 1;
+    const version = { ...fakeVersionWithDiff, id: headVersionId };
+    const store = configureStore();
+
+    store.dispatch(
+      versionsActions.loadVersionInfo({
+        version,
+        comparedToVersionId: baseVersionId,
+      }),
+    );
+
+    store.dispatch(
+      versionsActions.loadDiff({
+        addonId,
+        baseVersionId,
+        headVersionId,
+        version,
+      }),
+    );
+
+    // Simulate the beginning of a fetch.
```

Comment:
This comment does not seem needed as it repeats the name of the action dispatched below it. I'd rather see a comment saying that the action results in the file being loading, if any.